bulk-load/src/main/java/no/sikt/nva/data/report/api/etl/loader/BulkLoadHandler.java

```diff
@@ -0,0 +1,79 @@
+package no.sikt.nva.data.report.api.etl.loader;
+
+import static nva.commons.core.attempt.Try.attempt;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestStreamHandler;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpRequest.BodyPublishers;
+import java.net.http.HttpResponse.BodyHandlers;
+import no.sikt.nva.data.report.api.etl.loader.LoaderSpec.Builder;
+import nva.commons.core.Environment;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class BulkLoadHandler implements RequestStreamHandler {
+
+    public static final Logger logger = LoggerFactory.getLogger(BulkLoadHandler.class);
+    private static final String URI_TEMPLATE = "https://%s:%s/loader";
+    public static final String NEPTUNE_ENDPOINT = "NEPTUNE_ENDPOINT";
+    public static final String NEPTUNE_PORT = "NEPTUNE_PORT";
+    public static final String CONTENT_TYPE = "Content-Type";
+    public static final String APPLICATION_JSON = "application/json";
+    public static final String LOADER_IAM_ROLE = "LOADER_IAM_ROLE";
+    public static final String AWS_REGION = "AWS_REGION";
+    public static final String LOADER_BUCKET = "LOADER_BUCKET";
+    public static final int HTTP_OK = 200;
+    private final HttpClient httpClient;
+
+    public BulkLoadHandler(HttpClient httpClient) {
```

Comment:
You will probably need no-arg constructor

Reply:
Lol, I remember thinking "I'm going to forget this"